src/docfx.website.themes/common/common.js

```diff
@@ -118,3 +119,30 @@ function getOverrideTemplate(uid) {
     content += "\n";
     return content;
 }
+
+function mergeSeeAlso(model) {
+    var seealso = [];
```

Comment:
i'd like to merge children's seealso to parent's seealso section. namely, i think there should be only one SeeAlso section for a page.